doc/concepts/pfs/index.rst

```diff
@@ -0,0 +1,44 @@
+.. _pfs-overview:
+
+Pachyderm File System
+=====================
+
+Pachyderm File System (PFS) is a version-controlled file system
+in which you can store unlimited size files.
```

Comment:
"unlimited size files" does quite make sense. I believe our other PFS docs can give you wording for this.

Reply:
It probably makes sense to have a high-level summary of the PFS concepts here. 

The major concepts in Pachyderm are similar to Git at a high-level (might need additional clarification).

Data sets are organized into repositories. A repo can contain files and directories just like any normal file system. In addition, repos have a concept of commits, which are immutable snapshots of what the repo looked like at a given point in time.

... see below for details on each concept. <-- I know we don't like wording like this, just trying to get the point across.